services/core-api/app/api/incidents/models/mine_incident.py

```diff
@@ -88,19 +89,21 @@ class MineIncident(AuditMixin, Base):
         backref='mine_incidents',
         lazy='joined',
         uselist=False)
-    mine_incident_category_code = db.Column(
-        db.String, db.ForeignKey('mine_incident_category.mine_incident_category_code'))
-    mine_incident_category = db.relationship('MineIncidentCategory', lazy='joined')
 
     recommendations = db.relationship(
         'MineIncidentRecommendation',
         primaryjoin=
         "and_(MineIncidentRecommendation.mine_incident_id == MineIncident.mine_incident_id, MineIncidentRecommendation.deleted_ind==False)",
         lazy='selectin')
+
     documents = db.relationship('MineIncidentDocumentXref', lazy='joined')
     mine_documents = db.relationship(
         'MineDocument', lazy='joined', secondary='mine_incident_document_xref')
 
+    categories = db.relationship('MineIncidentCategoryXref', lazy='joined')
+    mine_categories = db.relationship(
+        'MineIncidentCategory', lazy='joined', secondary='mine_incident_category_xref')
+
```

Comment:
This now follows the same pattern used as the MineIncidentDocumentXref table as seen above.